projects/packages/external-media/.gitattributes

```
@@ -0,0 +1,18 @@
+# Files not needed to be distributed in the package.
+.gitattributes    export-ignore
+.github/          export-ignore
+package.json      export-ignore
+
+# Files to include in the mirror repo, but excluded via gitignore
+# Remember to end all directories with `/**` to properly tag every file.
+# /src/js/example.min.js  production-include
+src/build/**      production-include
+
+# Files to exclude from the mirror repo, but included in the monorepo.
+# Remember to end all directories with `/**` to properly tag every file.
+.gitignore        production-exclude
+changelog/**      production-exclude
+phpunit.xml.dist  production-exclude
+.phpcs.dir.xml    production-exclude
+tests/**          production-exclude
+.phpcsignore      production-exclude
```

Probably worth adding `babel.config.js` in there.

```suggestion
.gitignore        production-exclude
babel.config.js   production-exclude
changelog/**      production-exclude
phpunit.xml.dist  production-exclude
.phpcs.dir.xml    production-exclude
tests/**          production-exclude
.phpcsignore      production-exclude
```

---

Added by [2fec0bf](https://github.com/Automattic/jetpack/pull/41078/commits/2fec0bf52679df30dc48eaba957b60be6d2e8582)